RELEASE_PROCESS.md

```diff
@@ -2,38 +2,40 @@
 
 This document describes the procedure for releasing new versions of the core library.
 Most of the heavy lifting is actually done on GitHub Actions.
-All you have to do is trigger a GitHub release with the right tag.
+All you have to do is ensure the version in `allennlp/version.py` matches the target release version
+and then trigger a GitHub release with the right tag.
 
 > ❗️ This assumes you are using a clone of the main repo with the remote `origin` pointed
 to `git@github.com:allenai/allennlp.git` (or the `HTTPS` equivalent).
 
 The format of the tag should be `v{VERSION}`, i.e. the intended version of the release preceeded with a `v`.
 So for the version `1.0.0` release the tag will be `v1.0.0`.
 
-> ❗️ If this is a pre-release, such as `rc1`, it's not necessary to update `version.py`, as the pre-release suffix will be automatically taken from the tag. However, for this to work properly and to be consistent with our versioning conventions, make sure the suffix is separated with another `.`, for example
-> - ✅ `v1.0.0.rc1` = good,
-> - ❌ `v1.0.0rc1` = bad.
->
-> Note that the release on PyPI may not have the dot before the suffix since PyPI has its own weird versioning rules.
-
 To make things easier, start by setting the tag to an environment variable, `TAG`.
 Then you can copy and paste the commands below without worrying about mistyping the tag.
 
 ## Steps
 
-1.  Add the tag in git to mark the release:
+1. Update `allennlp/version.py` (if needed) with the correct version and then commit and push the changes with:
```

Comment:
We used to list the version in other places--such as the README.  Do we still do that and need to update the version elsewhere too?  It looks like we may have changed these links to point to the latest.

Reply:
I did a quick search and didn't find any other links that need to be updated, but I might have missed something